Formula/hasura-cli.rb

```diff
@@ -0,0 +1,34 @@
+class HasuraCli < Formula
+  desc "Command-Line Interface for Hasura GraphQL Engine"
+  homepage "https://hasura.io"
+  url "https://github.com/hasura/graphql-engine/archive/v1.2.2.tar.gz"
+  sha256 "59e15fe760794bb0ec69b62d0bf13d7925c2f595d78ee4121fc82fa67c36dbf9"
+
+  depends_on "go@1.13" => :build
```

Comment:
Could you please try if the formula builds with `go` rather than `go@1.13`?

Reply:
Yup, I'll try it out - the reason I did explicit 1.13 is because the `go.mod` file in the source specifies it.